### PR TITLE
Redesign tools page

### PR DESCRIPTION
**Pull request by sunny-viktoryia**
closes https://github.com/sparkletown/internal-sparkle-issues/issues/1698

TBD: fix the footer for the admin layout.

image
image
image

### CODE REVIEW

File: src/pages/Admin/ToolsPage/ToolsPage.tsx [hunk 1]
```diff
@@ -1,22 +1,20 @@
 import React, { useCallback, useState } from "react";
```
Comment (goran-peoski-work) [inline src/pages/Admin/ToolsPage/ToolsPage.tsx:1]: Empty undeleted file `src/pages/Admin/ToolsPage/ToolsPage.scss`

File: src/pages/Admin/ToolsPage/components/Tool/Tool.tsx [hunk 2]
```diff
@@ -1,20 +1,19 @@
 import React, { useState } from "react";
```
Comment (goran-peoski-work) [inline src/pages/Admin/ToolsPage/components/Tool/Tool.tsx:1]: Empty undeleted file `src/pages/Admin/ToolsPage/components/Tool/Tool.scss`

File: src/pages/Admin/ToolsPage/components/Tool/Tool.tsx [hunk 3]
```diff
@@ -43,26 +42,33 @@ export const Tool: React.FC<ToolOptions> = ({ tool, worldSlug }) => {
   );
 
   if (returnedData) {
-    return <tool.outputComponent {...returnedData} />;
+    return (
+      <ThreeColumnLayout>
+        <tool.outputComponent {...returnedData} />
```
Comment (goran-peoski-work) [inline src/pages/Admin/ToolsPage/components/Tool/Tool.tsx:47]: [Q] Does this name look OK?

File: src/pages/Admin/ToolsPage/scripts/analytics/AnalyticsOutput.tailwind.ts [hunk 4]
```diff
@@ -0,0 +1,2 @@
+export const a =
```
Comment (goran-peoski-work) [inline src/pages/Admin/ToolsPage/scripts/analytics/AnalyticsOutput.tailwind.ts:1]: [Q] Too short? `link` maybe?

File: src/pages/Admin/ToolsPage/scripts/makeWorldOwner/AddWorldOwnerOutput.tsx [hunk 5]
```diff
@@ -19,12 +18,23 @@ export const AddWorldOwnerOutput: React.FC<AddWorldOwnerOutputProps> = ({
 }) => (
   <div className="AddWorldOwnerOutput">
     {userEmail && <p>{userEmail} was successfully added as a world owner!</p>}
+    <br />
```
Comment (goran-peoski-work) [inline src/pages/Admin/ToolsPage/scripts/makeWorldOwner/AddWorldOwnerOutput.tsx:21]: [NOTE] Interesting container doesn't use flex with column direction, but a simple `<br />`.

**Review by goran-peoski-work — APPROVED** [review]
LGTM

File: src/pages/Admin/ToolsPage/ToolsPage.tsx (hunk 1 above)
Comment (sunny-viktoryia) [reply src/pages/Admin/ToolsPage/ToolsPage.tsx:1]: but it is already deleted. Looks like you haven't noticed. 🤔

File: src/pages/Admin/ToolsPage/components/Tool/Tool.tsx (hunk 3 above)
Comment (sunny-viktoryia) [reply src/pages/Admin/ToolsPage/components/Tool/Tool.tsx:47]: the name of the component is taken from the config of the tool. We can rename it if you have a better name in mind.

```ts
export type SelfServeScript = {
  name: string;
  description: string;
  functionLocation: string;
  arguments: ScriptArgument[];
  isBeta?: boolean;
  // NOTE: We want this component to be trully generic
  // eslint-disable-next-line @typescript-eslint/no-explicit-any
  outputComponent: React.FC<any>;
};
```

File: src/pages/Admin/ToolsPage/scripts/analytics/AnalyticsOutput.tailwind.ts (hunk 4 above)
Comment (sunny-viktoryia) [reply src/pages/Admin/ToolsPage/scripts/analytics/AnalyticsOutput.tailwind.ts]: renamed: https://github.com/sparkletown/sparkle/pull/2891/commits/2790ef994b91266445eca7e7fdf256dce9064186

**Pull request merged**